lib/units/auth/saml2.js

```diff
@@ -44,6 +43,17 @@ module.exports = function(options) {
     })
   }
 
+  if (options.saml.callbackUrl) {
```

Comment:
I did it on purpose, because it's more important to have clear intent and data, than one logic branch less, IMHO. Now the programmer's intent is very clear and data does not have unused values. If you still want me to change that, I can do it.